containers/ddev-router/Dockerfile

```diff
@@ -6,22 +6,60 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DOCKER_GEN_VERSION 0.7.4
 ENV DOCKER_HOST unix:///tmp/docker.sock
 
+# TARGETPLATFORM is Docker buildx's target platform (e.g. linux/arm64), while 
+# BUILDPLATFORM is the platform of the build host (e.g. linux/amd64)
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir /tmp/ddev && \
+    if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
+        printf '# jwilder/forego doesnt have an arm64 download so we resort to the original forego repo \n\
+FORGO_IS_TGZ="true" \n\
+FORGO_FILENAME="forego-stable-linux-arm64.tgz" \n\
+FORGO_URL="https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-arm64.tgz" \n\
+# jwilder/docker-gen doesnt have an arm64 download so we created a custom build \n\
+DOCKER_GEN_URL="https://github.com/dennisameling/docker-gen/releases/download/0.7.5/docker-gen-linux-arm64-0.7.5.tar.gz" \n\
```

Comment:
Shall we fork docker-gen for the arm64 build? What do you think?

Reply:
I forked and made you maintainer.

Reply:
https://github.com/drud/forego

Reply:
@rfay I actually need docker-gen instead of Forgo 😅

Reply:
Sorry, got docker-gen forked for you.

Reply:
Done: https://github.com/drud/docker-gen/releases/tag/0.7.5